### ISSUE TITLE
NVDA not reading the text of an EPUB file in Adobe DE v3.0

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4846]
**Reported by MichelleDawson on 2015-01-26 03:40**
I have tried all the previous fixes suggested for this issue in previous versions of all the pieces of software but not getting any results.
Working on Windows 8.1 pro.
Looking to use this combination as a solution for students who need a book I cannot get in an audio format, ie purchase ebook versions and have a screen reader read the book aloud.
Please note this did not work with JAWS either, so I have contacted both Adobe and the book supplier with this issue.

**Comment by ehollig** [comment]
Closing because we do not have a way to contact the original author and there is insufficient information to duplicate the issue.
